src/biogeochem/CNCIsoFluxMod.F90

```diff
@@ -1132,13 +1129,14 @@ subroutine CIsoFlux3(num_soilc , filter_soilc, num_soilp  , filter_soilp, &
                pp = col%patchi(cc) + pi - 1
                if (patch%active(pp)) then
                   do j = 1, nlevdecomp
-                     iso_cnveg_cf%m_c_to_litr_met_fire_col(cc,j) = iso_cnveg_cf%m_c_to_litr_met_fire_col(cc,j) + &
-                          ((iso_cnveg_cf%m_leafc_to_litter_fire_patch(pp)*lf_flab(ivt(pp)) &
+                     iso_cnveg_cf%m_c_to_litr_fire_col(cc,j,i_litr_min) = &
+                        iso_cnveg_cf%m_c_to_litr_fire_col(cc,j,i_litr_min) + &
+                          ((iso_cnveg_cf%m_leafc_to_litter_fire_patch(pp) * lf_f(ivt(pp),i_litr_min) &
```

Comment:
This is a case where metabolic litter was treated differently, so it remains outside the loop.

Reply:
Since this is outside the loop I also wonder if it shouldn't have a specific index set for it. So rather than i_litr_min, it would be "i_litr_meta"?

Reply:
We already had a private parameter i_met_lit in SoilBiogeochemDecompCascadeBGCMod.F90, so I moved it to clm_varpar and made it public.